CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 From v1.4, Orion requires Java 11. Orion on Java 8 is no longer supported. 
 
+## 1.5.1 
+
+## Additions and Improvements 
+
+* Moved Orion peer table to memory to support [high availability configurations](https://docs.orion.pegasys.tech/en/latest/HowTo/High-Availability/) [\#332](https://github.com/PegaSysEng/orion/pull/332)
```

Comment:
Actually, the table has always been in memory. What we did was add an option to store it in persistent storage :)

Reply:
Good point.  Updated